tools/settlement/bitflyer/upload.go

```diff
@@ -248,13 +249,23 @@ func setupSettlementTransactions(
 }
 
 func createBitflyerRequest(
+	ctx context.Context,
 	dryRun *bitflyer.DryRunOption,
 	token string,
 	settlementRequests []settlement.AggregateTransaction,
 ) (*bitflyer.WithdrawToDepositIDBulkPayload, error) {
 	set := []custodian.Transaction{}
 	sourceFrom := ""
+	maxAmountAny := ctx.Value(appctx.PayoutTxnMaxAmountCTXKey)
+	maxAmount, ok := maxAmountAny.(decimal.Decimal)
+	if !ok {
+		return nil, errors.New("provided max amount is not an integer")
```

Comment:
If there's any intent to make this unit-testable in the future, then replacing this with a pre-defined sentinel error (which can be tested against) would be good.